File: Python/2490.py
# 윷놀이 배(0), 등(1)
import sys, collections

def isResult(num):
    dic = {1 : 'A', 2 : 'B', 3 : 'C', 4 : 'D', 0 : 'E'}
    return dic[num]

f = lambda : sys.stdin.readline().rstrip()

for i in range(3):
    play = f().split()
    cnt = collections.Counter(play)
    print(isResult(cnt['0']))

File: Java/_2490.java
package Baekjoon;

import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.util.ArrayList;
import java.util.HashMap;
import java.util.List;

public class _2490 {

    public static void main(String[] args){

        try {

            BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

            for(int i = 0; i < 3; i++){

                List<String> zero = new ArrayList<String>();
                List<String> one = new ArrayList<String>();

                for(String temp : br.readLine().split(" ")){
                    if(temp.equals("0")) zero.add(temp);
                    else if(temp.equals("1")) one.add(temp);
                }
                System.out.println(isResult(zero.size()));
            }

        } catch (Exception e) { e.printStackTrace(); }
    }

    public static String isResult(int num){
        HashMap<Integer, String> map = new HashMap<>();
        map.put(1, "A");
        map.put(2, "B");
        map.put(3, "C");
        map.put(4, "D");
        map.put(0, "E");
        return map.get(num);
    }
}